bindings/python/CMakeLists.txt

```diff
@@ -15,12 +14,9 @@
 # Sets _ret to a list of python versions (major.minor) that use the same MSVC runtime as this build does
 # assumes MSVC was detected already
 # See https://en.wikipedia.org/wiki/Microsoft_Visual_C++#Internal_version_numbering
+# See https://devguide.python.org/#status-of-python-branches for supported python versions
 function(_get_compatible_python_versions _ret)
-	if (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 15 AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 16)
```

Comment:
Why?

Reply:
@zeule Python <3.5 has EOL'd. If support for older versions is still desired, I can add that later. To make things easier though, I'd like to get everything working first with just Python 3 (see my concerns in the OP), and then go from there.